tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_dsds_cell_data.py

```diff
@@ -17,26 +17,21 @@ def test_cell_data_for_two_sims(self):
               
         settings = Settings(self.marionette)
         settings.launch()
-
-        # Open cell data settings
         cell_and_data_settings = settings.open_cell_and_data_settings()
 
         # verify that a carrier is displayed
         self.assertTrue(len(cell_and_data_settings.carrier_name) > 0)
```

Comment:
Is it possible to change this assertion to:
``` Python
self.assertNotEqual(cell_and_data_settings.carrier_name, '')
```

This would report the displayed string if the test fails.